### DISCUSSION TITLE
[PrimeIcons] How to change thickness of PrimeIcon in p-checkbox

### DISCUSSION THREAD

**Discussion by GiangNguyenDev** [primefaces/community#101] [PrimeNG] [answered]
Hi,

I tried to change stroke-width property of svg in p-checkbox but it doesn't work. I also don't find any configuration setting in PrimeNG documentation for this purpose. Can you help me out? Thanks.

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Unfortunately there isn't an input property to customize stroke width of SVG icons in PrimeNG. However you can achieve it by using custom CSS as shown in the example: https://stackblitz.com/edit/a17srh?file=src%2Fapp%2Fdemo%2Fcheckbox-basic-demo.scss,src%2Fapp%2Fdemo%2Fcheckbox-basic-demo.ts

The reason why stroke-width didn't work is, you have to assign a color first. Hope it helps.

**Reply by GiangNguyenDev** [reply]
Thank you for your promt response. Yes, it works with stroke color. You helped me again to resolve an issue 🙂.

**Reply by cetincakiroglu** [reply]
Happy to help :)
